conf/application.conf

```diff
@@ -78,8 +78,6 @@ play.http.router=prod.Routes
 # The below controllers are the default exceptions to this rule.
 
 controllers {
-    # 300 is the default, you may need to change this according to your needs
-    confidenceLevel = 300
```

Comment:
I don't what this config is and how is it used and therefore removed it, let me know if this has any significance